notebooks/official/automl/sdk_automl_text_sentiment_analysis_online.ipynb

```diff
@@ -33,18 +33,18 @@
         "\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #17.    if delete_bucket is True:
</pre><p>We should add <code>or os.getenv("IS_TESTING") is True</code> to avoid creating extra buckets in the CI process.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/293/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='57'/>